designs/index.md

```diff
@@ -14,27 +14,37 @@ changes as designs are implemented.
 Always go to the [documentation]({{ site.docs_site_url }}) for descriptions of
```

Comment:
Currently, the levels of headings is a little erratic.  perhaps instead of starting with "here's a link to the proposal repo" without describing the context we should start off with what are design docs and why do we need them. 
Thoughts?

Reply:
Changed headers throughout :)